core/src/main/scala/kafka/coordinator/transaction/TransactionStateManager.scala

```diff
@@ -557,6 +557,7 @@ class TransactionStateManager(brokerId: Int,
           loadingPartitions.remove(partitionAndLeaderEpoch)
 
           transactionsPendingForCompletion.foreach { txnTransitMetadata =>
+            info(s"Sending txn markers for $txnTransitMetadata after loading partition $partitionId")
```

Comment:
I appreciate this log as well, but I wonder if we could combine it into a single log message.